v3/src/index.tsx

```diff
@@ -1,3 +1,6 @@
+// https://www.npmjs.com/package/@welldone-software/why-did-you-render
+// uncomment the line below to enable why-did-you-render (for development only)
```

Comment:
Could there be just a wee bit more documentation about how to use it? E.g. what if I'm only concerned about why one particular component renders?

Reply:
Configuration options and syntax are detailed here: https://github.com/welldone-software/why-did-you-render#readme.   In particular: [Tracking Components](https://github.com/welldone-software/why-did-you-render#options) and [Configuration Options](https://github.com/welldone-software/why-did-you-render#options)